modules/javafx.graphics/src/main/native-glass/monocle/android/nativeBridge.c

```diff
@@ -153,7 +157,19 @@ void androidJfx_requestGlassToRedraw() {
         GLASS_LOG_WARNING("we can't do this yet, no monocle_repaintAll\n");
         return;
     }
-    (*javaEnv)->CallStaticVoidMethod(javaEnv, jMonocleWindowManagerClass, monocle_repaintAll);
+    if (androidWindow == NULL) {
+        GLASS_LOG_WARNING("we can't do this yet, no androidWindow\n");
+        return;
+    }
+    int32_t width = ANativeWindow_getWidth(androidWindow) / androidDensity;
+    int32_t height = ANativeWindow_getHeight(androidWindow) / androidDensity;
+    jobject screen = (*javaEnv)->NewObject(javaEnv, jScreenClass, screen_init,
+        (jlong) androidWindow, 24,
+        0, 0, (jint) width, (jint) height,
+        0, 0, (jint) width, (jint) height,
+        0, 0, (jint) width, (jint) height,
+        100, 100, (jfloat) 1, (jfloat) 1, androidDensity, androidDensity);
```

Comment:
might be good to replace the `100` with e.g. `dpi` and assign a constant value (100) at the top of this file to it, in case we later want to retrieve that in a more dynamic way.